src/Core/Geographic/Coordinates.js

```diff
@@ -10,6 +10,8 @@ import mE from '../Math/MathExtended';
 import Ellipsoid from '../Math/Ellipsoid';
 
 proj4.defs('EPSG:4978', '+proj=geocent +datum=WGS84 +units=m +no_defs');
+// used in some protocol
+proj4.defs('CRS:84', proj4.defs('EPSG:4326'));
```

Comment:
can you add comment?
```js
// CRS:84 is equivalent to EPSG:4326 - ie, basic WGS84 degrees.
```

Reply:
I don't see the point of this comment. This line is pretty self-explanatory.

Reply:
not for the WGS84

Reply:
I see what you mean, but I'm not sure this is the place to explain CRS.

Reply:
I had to watch on the internet for explanations CRS:84, so it's a good place to explain to the developers. Why it makes sense to add this equivalence because we try to support wgs84 in itowns. It's better to be generous in commentary and it'sn't much to add

Reply:
ok